infer/inference.cc

```diff
@@ -11,6 +11,17 @@
 using namespace std;
 namespace sorbet::infer {
 
+const std::optional<core::ErrorClass> Inference::errorClassForUntyped(const core::GlobalState &gs, core::FileRef file) {
+    if (file.data(gs).strictLevel < core::StrictLevel::Strong) {
+        if (file.data(gs).isOpenInClient()) {
+            return core::errors::Infer::UntypedValueInformation;
+        } else {
+            return std::nullopt;
+        }
+    }
+    return core::errors::Infer::UntypedValue;
+}
```

Comment:
This function isn't used outside of this file, right?

Let's make it a helper function in an anonymous namespace:

```suggestion
namespace {

const std::optional<core::ErrorClass> errorClassForUntyped(const core::GlobalState &gs, core::FileRef file) {
    if (file.data(gs).strictLevel < core::StrictLevel::Strong) {
        if (file.data(gs).isOpenInClient()) {
            return core::errors::Infer::UntypedValueInformation;
        } else {
            return std::nullopt;
        }
    }
    return core::errors::Infer::UntypedValue;
}

} // namespace
```